gap/prop.gi

```diff
@@ -208,6 +212,12 @@ function(D)
   return true;
 end);
 
+InstallTrueMethod(IsConnectedDigraph, IsStronglyConnectedDigraph);
```

Comment:
We tend to put true methods in the `.gd` file. In particular, we already have this true method in `prop.gd`, so please don't duplicate it here:

https://github.com/digraphs/Digraphs/blob/33c3b9c6197c6dd6c33e04e66fdd4bff7d85c987/gap/prop.gd#L84

Reply:
Nice!  Will remove.